students/joel_rhode/lesson_1/assignment/tests/test_unit.py

```diff
@@ -0,0 +1,127 @@
+"""Unit tests for the inventory management system."""
+
+
```

Comment:
Try adding these lines:

```
import sys
sys.path.append('./inventory_management')
```

Make sure you call _test_unit.py_ from it's parent directory, something like this:

`$ python -m unittest tests/test_unit.py`